src/tiles/list/tile-list-compact.ts

```diff
@@ -41,7 +41,9 @@ export class TileListCompact extends LitElement {
           .loggedIn=${this.loggedIn}
         >
         </image-block>
-        <div id="title">${DOMPurify.sanitize(this.model?.title ?? '')}</div>
+        <a href=${this.href} id="title"
```

Comment:
QUESTION: This "id" isn't used anywhere else, is it?

Reply:
It's used in the CSS below to style this element, but I think that's it.

Reply:
Oh, on a re-read, I think you might have been asking if it could collide with another `id` somewhere. Short answer is it's scoped to this component's shadow DOM, so shouldn't be an issue.